ArcGISRuntimeSDKQt_CppSamples/Scenes/CreateTerrainSurfaceFromLocalTile/CreateTerrainSurfaceFromLocalTile.qrc

```diff
@@ -0,0 +1,11 @@
+<RCC>
+    <qresource prefix="/Samples/Scenes/CreateTerrainSurfaceFromLocalTile">
+        <file alias="sample.json">CreateTerrainSurfaceFromLocalTile.json</file>        
+        <file>CreateTerrainSurfaceFromLocalTile.qml</file>
+        <file>CreateTerrainSurfaceFromLocalTile.h</file>
+        <file>CreateTerrainSurfaceFromLocalTile.cpp</file>
+		<file>main.qml</file>
```

Comment:
indentation. I think this must be off in the json template because it keeps cropping up

Reply:
Aye there is, I'll fix it in the template.